datalad/core/distributed/clone.py

```diff
@@ -571,16 +586,64 @@ def postclonecfg_annexdataset(ds, reckless, description=None):
     lgr.debug("Initializing annex repo at %s", ds.path)
     # Note, that we cannot enforce annex-init via AnnexRepo().
     # If such an instance already exists, its __init__ will not be executed.
-    # Therefore do quick test once we have an object and decide whether to call its _init().
+    # Therefore do quick test once we have an object and decide whether to call
+    # its _init().
     #
     # Additionally, call init if we need to add a description (see #1403),
     # since AnnexRepo.__init__ can only do it with create=True
     repo = AnnexRepo(ds.path, init=True)
     if not repo.is_initialized() or description:
         repo._init(description=description)
-    if reckless:
+    if reckless == 'auto':
         repo._run_annex_command('untrust', annex_options=['here'])
 
+    elif reckless == 'ephemeral':
+        # with ephemeral we declare 'here' as 'dead' right away, whenever
+        # we symlink origins annex. Because we want annex to copy to
+        # the ria remote to get availability info correct for an eventual
+        # git-push into the store
+        # this will cause stuff like this for a locally present annexed file:
+        # % git annex whereis d1
+        # whereis d1 (0 copies) failed
+        # BUT this works:
+        # % git annex find . --not --in here
+        # % git annex find . --in here
+        # d1
+
+        # we don't want annex copy-to origin
+        ds.config.set(
+            'remote.origin.annex-ignore', 'true',
+            where='local')
+
+        ds.repo._run_annex_command('dead', annex_options=['here'])
+
+        if check_symlink_capability(ds.repo.dot_git / 'dl_link_test',
+                                    ds.repo.dot_git / 'dl_target_test'):
+            # symlink the annex to avoid needless copies in an emphemeral clone
+            annex_dir = ds.repo.dot_git / 'annex'
+            origin_annex_url = ds.config.get("remote.origin.url", None)
+            if origin_annex_url:
+                try:
+                    # deal with file:// scheme URLs as well as plain paths
+                    # if origin isn't local, we have nothing to do
```

Comment:
comment nit-pick: I'm frequently tripped up by comments in this project that drop a period and treat a new line as a sentence separator.  As a reader, I would prefer to not have to guess from the context.  Like here, I'd guess that it should be "paths. If", but I'm not sure.